Conf/Package/notarization_tool.sh

```
@@ -2,17 +2,51 @@
 
 echo "${NOTARIZATION_KEY_P8}" | base64 --decode > "${KEY_FILE_PATH}"
 
-/usr/bin/xcrun notarytool submit "${2}" --wait -v \
-  --key "${KEY_FILE_PATH}" --key-id "${NOTARIZATION_KEY_ID}" --issuer "${NOTARIZATION_ISSUER_ID}"
+readonly CREDS=(
+  --key "${KEY_FILE_PATH}"
+  --key-id "${NOTARIZATION_KEY_ID}"
+  --issuer "${NOTARIZATION_ISSUER_ID}"
+)
+
+readonly SUBMIT_OUTPUT=$(/usr/bin/mktemp -t notarytool-submit)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)